browser/ui/webui/playlist_ui.cc

```diff
@@ -43,6 +80,7 @@ PlaylistUI::PlaylistUI(content::WebUI* web_ui, const std::string& name)
   auto* source =
       CreateAndAddWebUIDataSource(web_ui, name, kPlaylistGenerated,
                                   kPlaylistGeneratedSize, IDR_PLAYLIST_HTML);
+  source->AddString("bravePlaylistPlayerUrl", kPlaylistURL);
```

Comment:
leftover?

Reply:
Oh yeah, I didn't use load time data for this URL. removed.